pulsar-client-cpp/lib/auth/athenz/ZTSClient.cc

```diff
@@ -143,7 +143,7 @@ std::string ZTSClient::ybase64Encode(const unsigned char *input, int length) {
 char *ZTSClient::base64Decode(const char *input) {
     BIO *bio, *b64;
     size_t length = strlen(input);
-    char *result = (char *)malloc(length);
+    char *result = (char *)calloc(length, sizeof(char));
```

Comment:
This function is called once when client connects to broker.
if reconnection occurs, this function will not be called for 3600m from the previous connection because RoleToken fetched using this privatekey will be cached for 3600m(default).
So this method is not called frequently.

Sorry, already null terminated string is set to `result[strlen(result)]`.
 but, I'd like to initialize memory.